var/spack/repos/builtin/packages/r-crayon/package.py

```diff
@@ -13,9 +13,9 @@ class RCrayon(RPackage):
     New styles can also be created easily. This package was inspired by the
     'chalk' 'JavaScript' project."""
 
-    homepage = "https://cran.r-project.org/package=sourcetools"
-    url      = "https://cran.rstudio.com/src/contrib/crayon_1.3.4.tar.gz"
-    list_url = "https://cran.r-project.org/src/contrib/Archive/crayon"
+    homepage = "https://cloud.r-project.org/package=sourcetools"
+    url      = "https://cloud.r-project.org/src/contrib/crayon_1.3.4.tar.gz"
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/crayon"
 
     version('1.3.4', '77c7c2906c59a3141306d86c89ffc7d3')
     version('1.3.2', 'fe29c6204d2d6ff4c2f9d107a03d0cb9')
```

Comment:
Depends:	R (≥ 3.0.2)

Reply:
@adamjstewart I am not seeing the R constraint listed on the CRAN page or the DESCRIPTION file for r-crayon.

Reply:
Oh, maybe that's because it has the wrong homepage!

Reply:
That would do it. For some reason that got changed in 050b753.

Reply:
Fixed.